docs/front.html

```diff
@@ -0,0 +1,382 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+
+    <title>5' intro to RO-Crate </title>
+
+    <style type="text/css">
+      .jumbotron .logo { 
+        height: 7rem; 
+      }
+      .front-divider {
+        height: 3rem;
+        background-color: rgba(0, 0, 0, .1);
+        border: solid rgba(0, 0, 0, .15);
+        border-width: 1px 0;
+        box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
+      }
+    </style>
+
+  </head>
+  <body>
+    <main>
+      
+  <div class="jumbotron jumbotron-fluid m-2">
```

Comment:
I thought the jumbotron class was dropped in bootstrap 5?

Reply:
Right, https://getbootstrap.com/docs/5.0/examples/jumbotron/ does this 'manually' now.